shell/renderer/api/context_bridge/render_frame_context_bridge_store.cc

```diff
@@ -76,11 +78,7 @@ WeakGlobalPairNode::WeakGlobalPairNode(WeakGlobalPair pair) {
   this->pair = std::move(pair);
 }
 
-WeakGlobalPairNode::~WeakGlobalPairNode() {
-  if (next) {
```

Comment:
how... was this ever sensible?

cc @MarshallOfSound

Reply:
I think past me reasoned that deleting the head of a list should delete everything in the list without thinking about how this particular linked list was being used